client/blocks/inline-help/index.jsx

```diff
@@ -1,155 +0,0 @@
-import { Button, RootChild, Gridicon } from '@automattic/components';
-import { isWithinBreakpoint } from '@automattic/viewport';
-import classNames from 'classnames';
-import debugFactory from 'debug';
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import AsyncLoad from 'calypso/components/async-load';
-import { recordTracksEvent } from 'calypso/state/analytics/actions';
-import isHappychatOpen from 'calypso/state/happychat/selectors/is-happychat-open';
-import { showInlineHelpPopover, hideInlineHelpPopover } from 'calypso/state/inline-help/actions';
-import isInlineHelpPopoverVisible from 'calypso/state/inline-help/selectors/is-inline-help-popover-visible';
```

Comment:
Noting that the `getAdminHelpResults` could likely be moved to the global selectors directory now (`client/state/selectors/`) since `client/state/inline-help` is no longer a state tree.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/78433/commits/8258451d2832df12b2871d8f3994986fdc4a54d2 👍